knpu/ep3-webpack/caching-versioning.md

```diff
@@ -55,6 +55,8 @@ And this outputs just the first 6 characters.
 To control the CSS filenames, find the `plugins` section. For `ExtractTextPlugin`,
 add `useVersioning`. If it's on, use `[name].[contenthash:6].css`:
 
+> Here we need to use [md5:contenthash:hex:6] for webpack@4
```

Comment:
If I'm correct, we won't upgrade our code to Webpack 4 - we only want to add notes, right? Then, for example, because we're on Webpack 3 (in video and in code we use 3rd version), we need to copy/paste the content of 2b589790b0 code block and twaek it according to your comment, i.e. add a note like:

```
***TIP
% describe here why we need this change %. If you're on Webpack 4 - use:

\`\`\`javascript
const useVersioning = true;
// ...
const webpackConfig = {
    // ...
    plugins: [
        new ExtractTextPlugin(
            useVersioning ? '[name].[md5:contenthash:hex:6].css' : '[name].css'
        ),
    ],
};
\`\`\`
***
```

If I understand your fix right.

So, we can't create expandable code blocks like 2b589790b0 because all code blocks are based on the code behind the course, so to create a proper code block we'll need to upgrade course code that we don't want, we only need to add a note.